ui/src/app/views/account-creation-modals/component.tsx

```diff
@@ -44,14 +40,10 @@ export class AccountCreationResendModalReact extends
   }
 
   send() {
-    const request: ResendWelcomeEmailRequest = {
+    this.props.profileApi.resendWelcomeEmail({
       username: this.props.username,
       creationNonce: this.props.creationNonce
-    };
-    const args: FetchArgs = ProfileApiFetchParamCreator().resendWelcomeEmail(request);
-    fetch(fullUrl(args.url), args.options)
-      .then(handleErrors)
-      .catch(error => console.log(error));
```

Comment:
+1
As a general rule, any time you create a promise you should think about what happens if that promise rejects. The generated `fetch` code looks like it has some baked in logic to reject on non-2xx responses, so we should always have a `catch` clause to handle potential errors. Depending on whether it's an 'expected' error or not, that might mean handling it gracefully with a nice message, or just tossing up a generic error box of some sort.

Reply:
Yes, I agree we need to handle it. My intent with this PR was not to alter the actual behavior in this component. I can put the `console.log()` back in, but this is effectively the same as not handling it (and I think it may get logged anyways by Angular, I can test that). The `handleErrors` function I removed here simply threw an error for non-200 error codes (which the `profileApi` implementation now does for us).

Reply:
(Yeah, tbh I prefer a console.error over console.log anyway) If the profileApi implementation does that, then that's fine for now, and we should add better error handling to our backlog/let Karthik know)